src/pwncore/routes/ctf/start.py

```diff
@@ -60,7 +54,10 @@ async def start_docker_container(ctf_id: int, response: Response):
             "ctf_id": ctf_id,
         }
 
-    if await Container.filter(team_id=team_id).count() >= config.max_containers_per_team:  # noqa: B950
+    if (
+        await Container.filter(team_id=team_id).count()
+        >= config.max_containers_per_team
+    ):  # noqa: B950
```

Comment:
Is the noqa still needed?

Reply:
Very weird, when I locally tested noqa was required since black would split it up and flake8 would say that the binary operator should not be on a new line.

Reply:
Dude I swear it works half of the time and it just fails the other half.